src/Processors/Transforms/Streaming/AggregatingTransformWithSubstream.cpp

```diff
@@ -270,7 +270,7 @@ void AggregatingTransformWithSubstream::checkpoint(CheckpointContextPtr ckpt_ctx
         for (const auto & [id, substream_ctx] : substream_contexts)
         {
             assert(id == substream_ctx->id);
-            substream_ctx->serialize(wb, getVersion());
+            serialize(*substream_ctx, wb, getVersion());
```

Comment:
i feel like `substream_ctx->serialize(wb, ...)` is probably better than the current version. Don't know why we change it.